### PR TITLE
powerstate from phidget

### PR DESCRIPTION
**Pull request by benmaidel**
This is a workaround as long as the bms system is not working!

### CODE REVIEW

**Comment by floweisshardt** [comment]
as this is a cob4-1/msh specific and temporary fix we should NOT merge this into cob_driver (as it would add a dependency from cob_bms_driver to cob_phidgets)

please move script to msh repository

**Comment by benmaidel** [comment]
but the bms bug is a general bug! It does not only concern the msh project. cob4 is not working with the bms driver... Until the bms bug is fixed, this should be the workaround for the cob4.

**Comment by benmaidel** [comment]
i know, having an dependency to another package isn't that nice. But they are inside the same repo...

**Comment by benmaidel** [comment]
as discussed with @ipa-fmw I've moved the phidget power_state script to an extra package

File: cob_phidget_power_state/package.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>cob_phidget_power_state</name>
+  <version>0.0.0</version>
+  <description>The cob_phidget_power_state package</description>
+
+  <maintainer email="bnm@ipa.fhg.de">Benjamin Maidel</maintainer>
+  <autor email="bnm@ipa.fhg.de">Benjamin Maidel</autor>
+
+  <license>BSD</license>
```
Comment (floweisshardt) [inline cob_phidget_power_state/package.xml]: Please use LGPL

File: cob_phidget_power_state/package.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>cob_phidget_power_state</name>
+  <version>0.0.0</version>
```
Comment (floweisshardt) [inline cob_phidget_power_state/package.xml]: should be the same version as all other packages in cob_driver (0.6.7)

**Comment by floweisshardt** [comment]
merge together with https://github.com/ipa320/cob_robots/pull/496

File: cob_phidget_power_state/package.xml (hunk 2 above)
Comment (benmaidel) [inline cob_phidget_power_state/package.xml]: why is this? The `cob_phidget_power_state` pkg is brand new

Comment (floweisshardt) [inline cob_phidget_power_state/package.xml]: the release process unfortunatelly requires all packages within a repository to have the same version number :-(

**Comment by floweisshardt** [comment]
CR: OK

**Pull request merged**